_1_basics/src/main/java/code/_4_student_effort/FizzBuzz/FizzBuzz.java

```diff
@@ -0,0 +1,40 @@
+package code._4_student_effort.FizzBuzz;
+
+public class FizzBuzz {
+
+    final int max = 100;
```

Comment:
daca modifici max in 105, care este divizibil cu 3,5,7 atunci o sa afiseze in loc de 105 FizzBuzzRizz ?

Reply:
Nu o sa afiseze FizzBuzzRizz. Am inteles greseala. O sa incerc sa refac.